editorial_material/editorial_policies.tex

```diff
@@ -0,0 +1,84 @@
+\documentclass{article}
+\usepackage{graphicx}
+\usepackage{hyperref}
+
+\begin{document}
+
+\title{Editorial Policies of LiveCoMS}
+\author{Editorial Board of LiveCoMS}
+
+\maketitle
+
+\begin{abstract}
+The current draft list of editorial board policies for the Living
+Journal of Computational Molecular Science (LiveCoMS). These policies
+are not yet active, as they are still in draft.
+\end{abstract}
+
+\section{Policies on Submissions to LiveCoMS}
+
+\subsection{Policy on Licensing}
+
+LiveCoMS does not ask for or request any copyright transfer.
+
+However, in order to submit to LiveCoMS, authors must provide, at
+minimum, a license for LiveCoMS to publish the article and distributed
+it free of charge. We recommend that the authors release the article
+under an open source license such as \href{Creative Commons
+  Attribution}{https://creativecommons.org/licenses/by/4.0/},
+releasing the document for anyone to copy and redistribute the
+material in any medium or format, and remix, transform, and build upon
+the material for any purpose, even commercially.  Making it available
+to all of course makes it possible for LiveCoMS to publish it, and for
+the community to edit and contribute.
```

Comment:
Add in closing, `"We highly recommend the CC-BY license in order to ensure your work can reach and help the broadest audience possible, and suggest that when considering the appropriate license you [read this analysis](http://openaccess.ox.ac.uk/2013/06/13/cc-by-what-does-it-mean-for-scholarly-articles-3/)"`.

Reply:
Done.